services/rabbit/configs/rabbitmq.conf.j2

```diff
@@ -0,0 +1,19 @@
+{% set NODE_IXS = range(1, (RABBIT_CLUSTER_NODE_COUNT | int) + 1) -%}
```

Comment:
minor: can we sync this with how rabbit is configured in osparc-simcore, so that the backend dev's setup mimicks the prod one closely?

Reply:
ok thx! thats fine